app/move/middleware.js

```diff
@@ -29,8 +29,16 @@ module.exports = {
 
     try {
       const move = await req.services.move.getByIdWithEvents(moveId)
+      const journeys = await req.services.journey.getByMoveId(moveId)
+      journeys.shift()
```

Comment:
This `shift()` method is used as we already show the latest move in the identity bar. So when we render all the Journeys we remove the latest one, as it's already shown